src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -28,11 +28,14 @@ public class HttpClient : IHttpClient
         private readonly ICached<CookieContainer> _cookieContainerCache;
         private readonly ICached<bool> _curlTLSFallbackCache;
 
+        public bool forceCurl { get; set; }
```

Comment:
Is this purely for testing? Can we do something else rather than setting a property thats never otherwise used?